test/runtests.jl

```diff
@@ -18,11 +18,12 @@ DocMeta.setdocmeta!(
     recursive=true,
 )
 
-GROUP = get(ENV, "JULIA_SCT_TEST_GROUP", "All")
+GROUP = get(ENV, "JULIA_SCT_TEST_GROUP", "Core")
```

Comment:
FYI @gdalle, this changes the default local test suite to run `Core` instead of `All`.